docs/api/product_variants.rst

```diff
@@ -0,0 +1,756 @@
+| productId     | url attribute  | Id of product for which the variants should be displayed |
++---------------+----------------+----------------------------------------------------------+
+| code          | request        | **(unique)** Product variant identifier                  |
++---------------+----------------+----------------------------------------------------------+
+
+Example
+.......
+
+.. code-block:: bash
+
+    curl http://sylius.dev/api/v1/products/63/variants/342 \
+        -H "Authorization: Bearer MWExMWM0NzE1NmUyZDgyZDJiMjEzMmFlMjQ4MzgwMmE4ZTkxYzM0YjdlN2U2YzliNDIyMTk1ZDhlNDYxYWE4Ng" \
+        -H "Content-Type: application/json" \
+        -X PUT \
+        --data '
+            {
+                "translations":{
+                    "en__US": {
+                        "name": "Small mug"
+                    }
+                },
+                "channel_pricings": [
+                    {
+                        "price": "1243"
+                    },
+                    {
+                        "price": "342"
+                    }
+                ],
+                "tracked": true,
+                "on_hand": 5,
+                "tax_category": "other",
+                "shipping_category": "default",
+                "width": 5,
+                "height": 10,
+                "depth": 15,
+                "weight": 20,
+                "option_values": {
+                    "type" :"small"
+                }
+            }
+        '
+
+Example Response
+~~~~~~~~~~~~~~~~
+
+.. code-block:: text
+
+    STATUS: 204 No Content
+
+In order to perform a partial update, you should use a PATCH method.
+
+Definition
+^^^^^^^^^^
+
+.. code-block:: text
+
+    PATCH /api/v1/sylius.dev/api/v1/products/{productId}/variants/{id}
+
++-------------------------------------+----------------+----------------------------------------------------------+
+| Parameter                           | Parameter type | Description                                              |
++=====================================+================+==========================================================+
+| Authorization                       | header         | Token received during authentication                     |
++-------------------------------------+----------------+----------------------------------------------------------+
+| id                                  | url attribute  | Id of requested resource                                 |
++-------------------------------------+----------------+----------------------------------------------------------+
+| productId                           | url attribute  | Id of product for which the variants should be displayed |
++-------------------------------------+----------------+----------------------------------------------------------+
+| translations['locale_code']['name'] | request        | Name of product variant                                  |
```

Comment:
It would be nice to habe all possible parameters listed here and add a `optional` to description